src/spinner/spinner.style.ts

```diff
@@ -0,0 +1,32 @@
+import { css } from "lit";
+
+export default css`
+  :host {
+    box-sizing: border-box;
+  }
+
+  .spinner {
+    width: 20px;
```

Comment:
Please use tokens and default values (same as the comment for line 21 in this file)